orientdb/src/test/java/org/hibernate/ogm/datastore/orientdb/test/jpa/OrientDbQueryParamTest.java

```diff
@@ -43,11 +38,10 @@
  */
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
 @RunWith(Parameterized.class)
-public class OrientDbQueryParamTest {
+public class OrientDbQueryParamTest extends OgmJpaTestCase {
```

Comment:
About NPE .... hm ... Can You send me a full stack trace? I am not check the  test.